app/client/components/flowStartMultipleProductDetailHandler.tsx

```diff
@@ -23,25 +23,32 @@ import { minWidth } from "../styles/breakpoints";
 import { sans } from "../styles/fonts";
 import { Button } from "./buttons";
 import { CallCentreNumbers } from "./callCentreNumbers";
+import { NavItem } from "./nav";
 import { NoProduct } from "./noProduct";
-import { PageContainer } from "./page";
+import {
+  PageContainer,
+  PageHeaderContainer,
+  PageNavAndContentContainer
+} from "./page";
 import { CardDisplay } from "./payment/cardDisplay";
 import { DirectDebitDisplay } from "./payment/directDebitDisplay";
 import {
   ReturnToYourProductButton,
   RouteableStepProps
 } from "./wizardRouterAdapter";
 
-const PaymentTypeRenderer = (subscription: Subscription) => {
-  if (subscription.card) {
-    return <CardDisplay margin="0" {...subscription.card} inline />;
-  } else if (subscription.mandate) {
-    return <DirectDebitDisplay {...subscription.mandate} inline />;
-  } else if (subscription.payPalEmail) {
-    return <span>via PayPal</span>;
-  }
-  return null;
-};
+type NewLayoutWrapperFunc = (children: ReactElement | null) => ReactElement;
+interface ConditionalNewLayoutWrapperProps {
+  condition: boolean | undefined;
+  wrapper: NewLayoutWrapperFunc;
+  children: ReactElement | null;
+}
+const ConditionalNewLayoutWrapper = ({
```

Comment:
please can we avoid the word 'new' and instead use something the new style of wrapper - 'WithLeftNav' or something